libraries/matrix/impl/src/main/kotlin/io/element/android/libraries/matrix/impl/verification/RustSessionVerificationService.kt

```diff
@@ -82,8 +82,8 @@ class RustSessionVerificationService(
 
     override val isReady = isSyncServiceReady.stateIn(sessionCoroutineScope, SharingStarted.Eagerly, false)
 
-    override val canVerifySessionFlow = combine(sessionVerifiedStatus, isReady) { verificationStatus, isReady ->
-        isReady && verificationStatus == SessionVerifiedStatus.NotVerified
+    override val canVerifySessionFlow = sessionVerifiedStatus.map { verificationStatus ->
+        verificationStatus == SessionVerifiedStatus.NotVerified
```

Comment:
I understand that if the `verificationStatus` get the value `SessionVerifiedStatus.NotVerified`, it means that the sliding sync is ready, but I am not 100% sure.